_spinnaker_install_admin_guides/secrets-vault.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Secrets with Vault
+order: 152
+---
+
+{:toc}
+
+
+This document describes how to set up Spinnaker secrets in Hashicorp's Vault. In this example, we'll be using the default kv secret engine called `secret` and will be storing github credentials, a kubeconfig file and a java keystore for SAML SSO.
+
+
+## Authorization
+
+We currently support two methods of authenticating to your Vault server.
```

Comment:
of authentication with Vault servers
or
to authenticate with Vault servers
??